de.metas.vertical.pharma/src/main/java/de/metas/impexp/product/PharmaProductImportProcess.java

```diff
@@ -369,93 +365,86 @@ private void importPrices(@NonNull final I_I_Pharma_Product importRecord)
 
 	private void createKAEP(@NonNull final I_I_Pharma_Product importRecord)
```

Comment:
is there a place where we documented what KAEP, APU etc means?

Reply:
> is there a place where we documented what KAEP, APU etc means?

Those abbreviations afaiu are coming from the domain (i.e. the pharma data)..I'm not sure if we need to document their meaning..

Reply:
Yes, are coming from the pharma domain, Are very specific.